### PR TITLE
Clickhouse Docs

### PR DESCRIPTION
**Pull request by ayush-shah**
### Describe your changes:

Clickhouse markdown docs

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

**Review by OnkarVO7 — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/public/locales/en-US/Dashboard/QuickSight.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ In this section, we provide guides and references to use the QuickSight connecto
 
 # Requirements
 <!-- to be updated -->
-You can find further information on the Kafka connector in the [docs](https://docs.open-metadata.org/connectors/dashboard/quicksight).
+You can find further information on the Quicksight connector in the [docs](https://docs.open-metadata.org/connectors/dashboard/quicksight).
```
Comment (TeddyCr) [inline openmetadata-ui/src/main/resources/ui/public/locales/en-US/Dashboard/QuickSight.md:8]: Any more details on the type of permission a user would need? I believe the user's IAM policy should have certain access to extract metadata. Checking the metadata ingestion should give us some indication on the type of operations we perform.

Comment (ayush-shah) [reply openmetadata-ui/src/main/resources/ui/public/locales/en-US/Dashboard/QuickSight.md:8]: makes sense, let me add those permissions requirement here

File: openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/Clickhouse.md [hunk 2]
```diff
@@ -3,84 +3,89 @@
 In this section, we provide guides and references to use the Clickhouse connector.
 
 # Requirements
-<!-- to be updated -->
-You can find further information on the Kafka connector in the [docs](https://docs.open-metadata.org/connectors/database/clickhouse).
+You can find further information on the Clickhouse connector in the [docs](https://docs.open-metadata.org/connectors/database/clickhouse).
```
Comment (TeddyCr) [inline openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/Clickhouse.md:6]: Can you add more details around the permission needed to extract metadata as well as add additional information for the other workflows and the queries needed to grant the permissions.

You can see more info here https://github.com/open-metadata/OpenMetadata/blob/7f278c67fd5edb369a79697653f0c66ce8a769a7/openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/Redshift.md

File: openmetadata-ui/src/main/resources/ui/public/locales/en-US/Dashboard/QuickSight.md (hunk 1 above)
Comment (ayush-shah) [reply openmetadata-ui/src/main/resources/ui/public/locales/en-US/Dashboard/QuickSight.md:8]: I will resolve the same in https://github.com/open-metadata/OpenMetadata/pull/11179

**Review by OnkarVO7 — APPROVED** [review]

**Review by NiharDoshi99 — APPROVED** [review]

File: openmetadata-docs-v1/content/v1.0.0/connectors/database/clickhouse/airflow.md [hunk 3]
```diff
@@ -109,13 +135,16 @@ This is a sample config for Clickhouse:
...
 
 {% /codeInfo %}
 
 {% codeInfo srNumber=6 %}
 
-**scheme**: SQLAlchemy driver scheme options
+**scheme**: There are 2 types of schemes that the user can choose from.
+
+      **clickhouse+http**: Uses ClickHouse's HTTP interface for communication. Widely supported, but slower than native.
+      **clickhouse+native**: Uses the native ClickHouse TCP protocol for communication. Faster than http, but may require additional server-side configuration. Recommended for performance-critical applications.
```
Comment (nahuelverdugo) [inline openmetadata-docs-v1/content/v1.0.0/connectors/database/clickhouse/airflow.md:147]:
```suggestion
**scheme**: There are 2 types of schemes that the user can choose from.
      - **clickhouse+http**: Uses ClickHouse's HTTP interface for communication. Widely supported, but slower than native.
      - **clickhouse+native**: Uses the native ClickHouse TCP protocol for communication. Faster than http, but may require additional server-side configuration. Recommended for performance-critical applications.
```
What about using a list?

Comment (nahuelverdugo) [reply openmetadata-docs-v1/content/v1.0.0/connectors/database/clickhouse/airflow.md:147]: Same comment for `cli.md`.

**Review by nahuelverdugo — DISMISSED** [review]

**Review by NiharDoshi99 — APPROVED** [review]

**Pull request merged**